pkg/providers/snow/reconciler/reconciler.go

```diff
@@ -66,6 +68,11 @@ func (r *Reconciler) ValidateMachineConfigs(ctx context.Context, log logr.Logger
 	return controller.Result{}, nil
 }
 
+func (r *Reconciler) CheckControlPlaneReady(ctx context.Context, log logr.Logger, clusterSpec *cluster.Spec) (controller.Result, error) {
```

Comment:
What calls this?

Reply:
The `Reconcile` method, in this file, line 48

Reply:
Ahh 🤦🏻